### ISSUE TITLE
How to reproduce the performance reported in your paper

### ISSUE THREAD

**Issue by slothfulxtx** [Asparagus15/GaussianShader#2]
Hi, authors,
     Thanks for your interesting work! When I tried to reproduce the performance on  Shiny Blender dataset and run your code, I found that some of your default arguments are not consistent with your paper. For example, the weighted hyperparameters of each loss terms. After I fixed these issues and made  the code runnable, I still cannot reproduce the performance you report in the paper. Could you tell me how to config the arguments for shiny blender? For examples, `brdf_dim = ?`

**Comment by cv-lab-x** [comment]
> Hi, authors, Thanks for your interesting work! When I tried to reproduce the performance on Shiny Blender dataset and run your code, I found that some of your default arguments are not consistent with your paper. For example, the weighted hyperparameters of each loss terms. After I fixed these issues and made the code runnable, I still cannot reproduce the performance you report in the paper. Could you tell me how to config the arguments for shiny blender? For examples, `brdf_dim = ?`

@slothfulxtx may set brdf_dim as same as sh_degree=3   [](https://github.com/Asparagus15/GaussianShader/blob/7e1931822ec963fd0fba64fcd7eb3ee0c731cd2a/gaussian_renderer/__init__.py#L148C24-L148C24)

**Comment by Asparagus15** [comment]
Sorry for the delay and thanks for the attention. Please check the latest update. I have uploaded an example data with training and testing instructions. The hyperparameter default values are also updated.

**Comment by ingra14m** [comment]
Hi, thanks for your great work and the clever idea of the short axis normal.

However, when I tried to replicate the results of the paper, I encountered some inconsistencies. I cloned the latest code, and my command for running it was shown below. 

```shell
python train.py -s ../data/refnerf/ball --eval -m output/ball_blender -w --brdf_dim 0 --sh_degree -1 --lambda_predicted_normal 2e-1 --brdf_env 512
```
This resulted in the following outcomes. 
![image](https://github.com/Asparagus15/GaussianShader/assets/63096187/98d6d8cb-e2b4-4df0-9bed-d8b30e264a3b)

Actually, for me, I was hoping to obtain a relatively perfect normal. It would be great if the author could point out the core idea behind the training of the normals, such as the learning rate, supervised loss, etc.
